deployment/articles-dxp/01-saml/02-setting-up-identity-provider.markdown

```diff
@@ -63,12 +63,13 @@ also by @product@'s permissions system.
     - The key length in bits (2048 is the default)
     - The key password
 
-    When you enter all the required information, click *Save*.
+    Click *Save*.
 
     When you create the certificate and private key, you also create a keystore
     if one doesn't already exist. As described above, this keystore has two
     storage options: file system storage (the default) and Documents and Media
-    storage. 
+    storage. By default, the certificate uses the SHA256 algorithm for
+    encryption and is fingerprinted and self-signed via MD5 and SHA1. 
```

Comment:
p.s. "self-signed via MD5 and SHA1." will be "self-signed via RSA and SHA256." in a build from a commit after 24/04/2019 (to include LPS-91918) . Beta 3 is missing it.

Reply:
Fixed in latest push.